org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/AbstractDesktop.java

```diff
@@ -318,4 +329,13 @@ protected void showHighlighted(Window win, String position)
     	}
     }
 
+	@Override
+	public void setPredefinedContextVariables(String predefinedVariables) {
+		this.predefinedContextVariables = predefinedVariables;
+	}
+
+	public String getPredefinedContextVariables() {
```

Comment:
the get and set should be protected instead, it is just for internal use, i.e not public api